### PR TITLE
Addition translation context for ships

### PR DESCRIPTION
**Pull request by muerteFR**
Added translation contexts for ships : 
"ship", "shipP", "class" and "subclass"

### CODE REVIEW

File: scripts/shipTemplates_StarFighters.lua [hunk 1]
```diff
@@ -133,7 +133,7 @@ template:setTubeSize(2, "small")
 template:setTubeDirection(1, 1):setWeaponTubeExclusiveFor(1, "HVLI")
 template:setTubeDirection(2,-1):setWeaponTubeExclusiveFor(2, "HVLI")
 
-variation = template:copy("ZX-Lindworm"):setLocaleName(_("ZX-Lindworm")):setModel("LindwurmFighterBlue"):setType("playership")
+variation = template:copy("ZX-Lindworm"):setLocaleName(_("shipP", "ZX-Lindworm")):setModel("LindwurmFighterBlue"):setType("playership")
```
Comment (daid) [inline scripts/shipTemplates_StarFighters.lua:136]: "ship**P**"

Comment (muerteFR) [reply scripts/shipTemplates_StarFighters.lua:136]: Yes, I put the context ShipP(layer) for the 18 player ships. To clarify more.
I tested this PR locally with ninja update_locale and everything works fine except for an issue already present on version 23112020: player ship names are not translated. (classes and subclasses yes but no the ship names)

Comment (muerteFR) [reply scripts/shipTemplates_StarFighters.lua:136]: And It's not just with fr.po, it's general. This is also not a problem with accents.

Comment (daid) [reply scripts/shipTemplates_StarFighters.lua:136]: shipP, why not just playerShip then?

Comment (muerteFR) [reply scripts/shipTemplates_StarFighters.lua:136]: Because I thought that the context shouldn't be too long.
Then I replace them in playerShip. :-)

Comment (muerteFR) [reply scripts/shipTemplates_StarFighters.lua]: Done

Comment (muerteFR) [reply scripts/shipTemplates_StarFighters.lua]: @daid , is there a contraindication to validating this PR? Thanks

**Pull request merged**